tests/resources/execution_module/page_steps_builder.js

```diff
@@ -1,42 +1,88 @@
-function getInputValue(step){
+function getInputValue(step) {
     return step.value
 }
+function inputValueIsFeeders(value, FEEDERS) {
+    var regexp1 = /^\${(.*?)}$/
```

Comment:
Doublecheck regular expressions. Seems like they are the same.

Reply:
code duplication was fixed